src/Client/Adapter/Curl.php

```diff
@@ -209,6 +209,13 @@ public function connect($host, $port = 80, $secure = false)
             }
         }
 
+        if (isset($this->config['sslcafile']) && $this->config['sslcafile']) {
```

Comment:
> @mlocati why you are against using an Exception for this case?

@ezimuel Because I love to keep everything as simple as possible: if someone sets `sslverifypeer` to false, he excepts that `CURLOPT_SSL_VERIFYPEER` is set to false, for any value of the other parameters